### DISCUSSION TITLE
how to generate CAPTCHA

### DISCUSSION THREAD

**Discussion by ghost** [emilk/egui#2181] [Q&A]
Can egui generate CAPTCHA to prevent DDoS and spam?

**Comment by willser** [comment]
Egui provide function to load image,you could load image from `&[u8]`.So you can get it from a url or created by other lib.
https://github.com/emilk/egui/blob/5e44c13b481d8bdcadde977f04ed3cd078975359/crates/epaint/src/image.rs#L65-L94
